src/Microsoft.ML.StandardTrainers/Standard/LogisticRegression/LbfgsPredictorBase.cs

```diff
@@ -48,7 +54,7 @@ public abstract class OptionsBase : TrainerInputBaseWithWeight
             public float OptmizationTolerance = Defaults.OptimizationTolerance;
 
             /// <summary>
-            /// Number of previous iterations to remember for estimate of Hessian.
+            /// Number of previous iterations to remember for estimate of Hessian. Lower values mean faster but less accurate estimates.
```

Comment:
> for estimate of Hessian [](start = 57, length = 24)

ditto

Reply:
done

---
In reply to: [268038103](https://github.com/dotnet/machinelearning/pull/3059#discussion_r268038103) [](ancestors = 268038103)